spec/features/issue_pages/datatable_spec.rb

```diff
@@ -1,8 +1,41 @@
-# let(:default_columns) { ['Title', 'Created', ...] }
-# let(:hidden_columns) { ['Description', 'Extra', ...] }
-# let(:filter) { { keyword:'keyword', number_of_rows: 1 } }
-shared_examples 'a DataTable' do
-  describe 'column visibility', js: true do
+require 'rails_helper'
+
+describe 'issue table', js: true do
+  subject { page }
+
+  let(:issue) { issues[0] }
+  let(:issues) do
+    [
+      create(
+        :issue,
+        text: "#[Title]#\nIssue1\n\n#[Risk]#\nHigh\n\n#[Description]#\nn/a",
+        node: current_project.issue_library
+      ),
+      create(:issue, node: current_project.issue_library)
+    ]
+  end
+
+  let(:tags) do
+    Tag::DEFAULT_TAGS.map do |tag|
+      if defined?(Dradis::Pro)
```

Comment:
I feel like we can improve this. Is it possible to update the factory instead and have it accept a project association even in CE? Seeing as `current_project` exists in both editions, the ideal code here would be: `create(:tag, name: tag, project: current_project)`. I do see that this piece of code is used elsewhere in the test suite so I'm fine with moving forward with this.

Reply:
> Is it possible to update the factory instead and have it accept a project association even in CE?
Yeah we can use transient attributes https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#transient-attributes